templates/communities.html

```diff
@@ -1,23 +1,60 @@
 {% extends 'components/_base.html' %}
 {% block title %}Open Source Communities{% endblock %}
 {% block content %}
-    <div id="contentWrap">
-        {% for community in communities %}
-        <section class="communityBox auto-center" id="{{ community.tag }}">
-            <div class="container">
-                <div class="row">
-                    <div class="col-lg-12 headerContainer col-centered">
-                        <h1 class="section-header text-center">{{ community.title }}</h1>
-                    </div>
+    <main id="contentWrap" class="communitiesContentWrap">
+        <div class="gridWrap">
+            <div class="grid-heading">
+                <div class="bottom-border center">
+                    <h1 class="section-header text-center" >Communities</h1>
                 </div>
-                <div class="row">
+            </div>
+            {% for community in communities %}
+            <div class="card">
+                <div class="cardLogo">
                     {%- if community.logo|length > 0 %}
-                    <img class="logo-med" src="{{ community.logo }}"/>
+                        <img class="logo-med communities" src="{{ community.logo }}"/>
+                    {%- else%}
+                        <h1 class="section-header text-center">{{ community.title }}</h1>
+                    {%- endif %}
+                </div>
+                <div class="cardBody">
+                    {{ community.desc }}
+                </div>
+                <div class="cardFooter">
+                    {%- for link in community.links %}
+                        <a href="{{ link.url }}" target="_blank">
+                            <i class="fa fa-{{ link.icon }}"></i>
+                        </a>
+                    {%- endfor %}
+                </div>
+            </div>
+            {% endfor %}
+            <div class="grid-heading">
+                <div class="bottom-border center">
+                    <h1 class="section-header text-center" >Conferences</h1>

```

Comment:
Should we change this to `Conferences & Open Source Programs` . I think that'll be better if we really want to add conference and other programs to this section. 

CC: @vipulgupta2048

Reply:
Change applied
![screencapture-0-0-0-0-8000-communities-html-2020-06-11-16_07_43](https://user-images.githubusercontent.com/49009626/84447739-b893f000-abfd-11ea-8c8d-c73a0a7b3685.png)